extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -35,20 +34,25 @@ public function index() {
 		wfProfileIn( __METHOD__ );
 
 		// Since this "Community_App" article won't be found, we need to manually say it's okay so that it's not a 404.
-		$this->response->setCode( self::RESPONSE_OK );
+		$this->response->setCode( static::RESPONSE_OK );
 
 		// Pull in the app-configuration (has data for all apps)
 		$appConfig = [];
 		$memcKey = wfMemcKey( static::$CACHE_KEY, static::$CACHE_KEY_VERSION );
 		$response = $this->wg->memc->get( $memcKey );
 		if ( empty( $response ) ){
```

Comment:
You could simplify this code using: https://github.com/Wikia/app/blob/f139fa71cc912c459d364f32e1de40c55c7b950a/includes/wikia/WikiaDataAccess.class.php#L52-L52

See examples: https://github.com/Wikia/app/blob/f139fa71cc912c459d364f32e1de40c55c7b950a/includes/wikia/WikiaDataAccess.class.php#L308-L397